digest/index.test.js

```diff
@@ -51,6 +51,16 @@ describe('digest', function(){
           expect(phase.skills).to.not.haveDuplicates()
         })
       })
+
+      it('should link to existing modules', function(){
+        const modules = Object.values(this.digest.modules)
+          .map(module => module.directoryName)
+
+        Object.values(this.digest.phases)
+          .forEach(phase => {
+            expect(modules).to.include.members(phase.modules)
```

Comment:
I like it! I just wish it had a better error message. I would do something like this:

```js
it('should link to existing modules', function(){
  const allModuleIds = Object.values(this.digest.modules)
    .map(module => module.id)
    .sort()

  const moduleIdsInPhases = Object.values(this.digest.phases)
    .map(phase => phase.modules)
    .reduce((a, b) => a.concat(b))
    .sort()

  const missingModuleIds = moduleIdsInPhases.filter(moduleId =>
    !allModuleIds.includes(moduleId)
  )

  if (missingModuleIds.length > 0)
    throw new AssertionError(
      `The following modules referenced by phases are missing:`+
      `\n    - ${missingModuleIds.join("\n    - ")}`
    )
})
```